### PR TITLE
Authenticate unrestricted file deletion

### PR DESCRIPTION
**Pull request by Falence**
Authenticates user before deleting file

### CODE REVIEW

File: patient_portal/import_template.php [hunk 1]
```diff
@@ -27,9 +27,17 @@
     file_put_contents($_POST['docid'], $_POST['content']);
     exit(true);
 } else if ($_POST['mode'] == 'delete') {
-    unlink($_POST['docid']);
-    exit(true);
+
+    // authenticate
+    $userAuthorized = $_SESSION['userauthorized'];
```
Comment (robbyoconnor) [inline patient_portal/import_template.php]: Incorrect.

**Comment by muarachmann** [comment]
Why so much commits for just a few lines? @Falence are you having issues with git... you can always learn from our awesome community

**Comment by aethelwulffe** [comment]
Wow.  Way to let 'em off easy.  This isn't an IDE automated response
where it is OK to just say "FAILURE:  Line 1.".:-!

**Comment by robbyoconnor** [comment]
> Wow.  Way to let 'em off easy.  This isn't an IDE automated response where it is OK to just say "FAILURE:  Line 1.".:-!

**Comment by robbyoconnor** [comment]
> Wow.  Way to let 'em off easy.  This isn't an IDE automated response where it is OK to just say "FAILURE:  Line 1.".:-!

@aethelwulffe wait, are you saying I should have been harsher than I was? I'm confused.

**Comment by robbyoconnor** [comment]
I'm going to close this.

**Pull request closed without merging**